lib/ignore.js

```diff
@@ -56,7 +56,7 @@ function ignore(dir, options) {
     })
     .then(function(lines) {
       // Add "manifest.json" to always exclude it from the xpi.
```

Comment:
To make it extra clear, you should update this comment mentioning the embedded case.